src/main/java/randoop/reflection/OperationModel.java

```diff
@@ -466,10 +467,14 @@ private void addClassTypes(
       mgr.add(new ClassLiteralExtractor(this.classLiteralMap));
     }
 
+    // Need to load in instrumented versions of all classes under test first.
```

Comment:
Should the changes in this class depend on `GenInputsAbstract.enable_bloodhound`, or should they be unconditional as they are now?